widgets/hits-per-page-selector/hits-per-page-selector.js

```diff
@@ -41,14 +41,14 @@ function hitsPerPageSelector({
   return {
     init: function(state) {
       let isCurrentInOptions = reduce(options, function(res, option) {
```

Comment:
Yup or you can use native JS Array.some which does that: stop when one of the condition is true.

`options.some(matchingHitsPerPage)`